src/lib/isMobilePhone.js

```diff
@@ -100,7 +100,7 @@ const phones = {
   'fr-BF': /^(\+226|0)[67]\d{7}$/,
   'fr-BJ': /^(\+229)\d{8}$/,
   'fr-CD': /^(\+?243|0)?(8|9)\d{8}$/,
-  'fr-CM': /^(\+?237)6[0-9]{8}$/,
+  'fr-CM': /^(\+?237)(6|2)[0-9]{8}$/,
```

Comment:
This change is not needed as far as I can see. A number starting with `2` is not for mobile phone numbers, only fixed numbers (which we don't validate here). We can improve this regular expression to make it more strict though.

The latest ITU information (which is my primary source) is from 2014, so it might be outdated;
https://www.itu.int/oth/T0202000024/en

Reply:
I had some work done for a client with a modem who's number started with 2. The modem uses LTE for connectivity so would that be a fixed or mobile classification?